app/controllers/shipit/webhooks_controller.rb

```diff
@@ -43,7 +43,20 @@ def event
 
     def repository_owner
       # Fallback to the organization sub-object if repository isn't included in the payload
-      params.dig('repository', 'owner', 'login') || params.dig('organization', 'login')
+      owner = params.dig('repository', 'owner', 'login') || params.dig('organization', 'login')
+
+      if owner.nil?
+        # Fallback to less common locations
+        if params.dig('installation', 'account', 'type') == 'Organization'
```

Comment:
Do we even care about these events?

Couldn't we refactor a bit to first check wether the event has a handler before going down that route?

Reply:
Agree on the need for a refactor. I don't want to tackle that in this PR though.

I like what you proposed @ajshepley. I went with that approach, and also dropped the head response for if `repository_owner` cannot be found. It didn't work as expected, and now for events that we do support we want to be able to handle them successfully (i.e. detecting the organization so we can choose the right Github app).

Reply:
> I can see us moving the repository_owner parsing as a trait of Shipit::Webhooks::Handler or something like that, so that we don't build an endlessly bigger method here, but I don't know if that'd be necessary right now.

:100: The handler in the future should be authoritative for parsing its payload and determining the organization as the payload structure will be known.